python/lsst/ts/idl/enums/HVAC.py

```diff
@@ -0,0 +1,139 @@
+# This file is part of ts_idl.
+#
+# Developed for the Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["DeviceId", "DEVICE_GROUPS", "DEVICE_GROUP_IDS"]
+
+from enum import Enum
+
+
+class DeviceId(Enum):
```

Comment:
Please consider using `IntEnum` here, so you can more easily handle the integer values that appear in SAL/DDS messages. I believe the main advantage is that you can assign enums directly to int DDS message fields, e.g.:
```
self.evt_foo.set_put(bar=DeviceId.generalP01)
```
instead of
```
self.evt_foo.set_put(bar=DeviceId.gneralP01.value)
```

Reply:
I did consider that but am not sure anymore why I didn't do that in the first place. I have changed it now.